mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -281,6 +284,26 @@ public static void applyAuthentication(WebView webView, Connection connection, S
             webView.setHttpAuthUsernamePassword(Util.getHostFromUrl(url), "",
                     connection.getUsername(), connection.getPassword());
         }
+
+        webView.getSettings().setDomStorageEnabled(true);
+        webView.getSettings().setJavaScriptEnabled(true);
+        webView.getSettings().setSupportMultipleWindows(true);
+
+        webView.setWebChromeClient(new WebChromeClient() {
+            @Override
+            public boolean onCreateWindow(WebView view, boolean dialog, boolean userGesture,
+                    android.os.Message resultMsg) {
```

Comment:
As you import it above you don't need the fully qualified name here.

Reply:
Fixed